fwffr.py

```diff
@@ -26,6 +26,9 @@ def iteritems(d, **kw):
 
 class FixedLengthError(ValueError):
     """ Base class for parsing errors """
+    
+    def __init__(self, message=None):
+        self.message = message
 
     def __str__(self):
         return self.message
```

Comment:
This class provides no additional functionality, so we should remove its body entirely so it defers to `ValueError` for all behavior.

```suggestion
```